gulpfile.js

```diff
@@ -0,0 +1,40 @@
+import gulp from "gulp";
+
+import { path } from "./gulp/config/path.js";
+
+import { plugins } from "./gulp/config/plugins.js";
+
+global.app = {
+	isBuild: process.argv.includes("--build"),
+	isDev: !process.argv.includes("--build"),
+	path: path,
+	gulp: gulp,
+	plugins: plugins
+}
+
+import { copy } from "./gulp/tasks/copy.js";
```

Comment:
looks like you don't have folder called "files", so you don't need this task

Reply:
Done